kafka/sarama.go

```diff
@@ -6,16 +6,11 @@ import (
 	"strings"
 
 	"github.com/Shopify/sarama"
-	fromuri "github.com/artyomturkin/go-from-uri"
+	"github.com/artyomturkin/go-from-uri"
 )
 
 // NewSaramaConfig build new sarama config from URL string.
-func NewSaramaConfig(connectionURL string) ([]string, *sarama.Config, error) {
-	u, err := url.Parse(connectionURL)
-	if err != nil {
-		return nil, nil, err
-	}
-
+func NewSaramaConfig(u *url.URL) ([]string, *sarama.Config, error) {
```

Comment:
Do not change interface, it'll break other packages

Reply:
> Do not change interface, it'll break other packages

ok